src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -4,24 +4,24 @@
 import edu.kis.powp.jobs2d.LoggerDriver;
 
 /**
- * Driver manager provides means to setup the driver. It also enables other
- * components and features of the application to react on configuration changes.
+ * Driver manager provides means to setup the driver. It also enables other components and features
+ * of the application to react on configuration changes.
  */
 public class DriverManager {
 
-	private Job2dDriver currentDriver = new LoggerDriver();
+  private Job2dDriver currentDriver = new LoggerDriver();
 
-	/**
-	 * @param driver Set the driver as current.
-	 */
-	public synchronized void setCurrentDriver(Job2dDriver driver) {
-		currentDriver = driver;
-	}
+  /**
+   * @param driver Set the driver as current.
+   */
+  public synchronized void setCurrentDriver(Job2dDriver driver) {
+    currentDriver = driver;
+  }
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.